pysam/libcbcf.pyx

```diff
@@ -1184,6 +1185,15 @@ cdef inline bcf_sync_end(VariantRecord record):
         bcf_info_set_value(record, b'END', record.ptr.pos + record.ptr.rlen)
 
 
+cdef inline bint has_symbolic_allele(VariantRecord record):
+    # Check if alternate allele is symbolic
+    if record.ptr.n_allele >= 2:
```

Comment:
The code below has to be a loop, since any of the alt alleles could be symbolic.